bedrock/thunderbird/templates/thunderbird/base.html

```diff
@@ -0,0 +1,31 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base.html" %}
+
+{% block facebook_id %}348169218537151{# facebook.com/Thunderbird #}{% endblock %}
+{% block twitter_id %}mozthunderbird{% endblock %}
+
+{% block page_image %}{{ static('img/thunderbird/thunderbird-256.png') }}{% endblock %}
+{% block page_favicon %}{{ static('img/thunderbird/favicon.ico') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/thunderbird/favicon-196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/thunderbird/ios-icon-180.png') }}{% endblock %}
+
+{% block body_class %}sky{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox' %}
+{% endblock %}
+
+{% block site_header_nav %}
+  {% include 'thunderbird/includes/top-menu.html' %}
+{% endblock %}
+
+{% block site_header_logo %}
+<h2><a href="{{ url('thunderbird.index') }}">{{ high_res_img('img/thunderbird/template/logo-wordmark.png', {'alt': 'Thunderbird', 'width': '260', 'height': '72'}) }}</a></h2>
```

Comment:
Seeing that we merged the PR to change `high_res_img` etc. you no longer need 'img/...'